lab-steven/lib/text-read.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+
+// module.exports = exports = {};
+
+
+
+// exports.readDone = [];
+
+module.exports.textReader = function(filename1, filename2, filename3, callback) {
+  let fileReadDone = [];
+  fs.readFile(`${__dirname}/../data/${filename1}.txt`, function(err, data){
+    if(err) throw err;
+    // console.log(data.toString('hex', 0, 8), filename);
```

Comment:
try to remove your commented out code in the submissions. it's nitpicky, but Scott wants to get in the habit of removing these from submitted code. i personally don't mind because i like to see my own thought process as things get executed.